### PR TITLE
[Project A][Win11][jedkohjk] Upgrade to Java 17

### PR DESCRIPTION
**Pull request by jedkohjk**
OS: Windows 11 (asked a friend about Mac)
JDK Distribution: Oracle
Link to JAR file: [https://github.com/jedkohjk/AB3-J17/releases/download/Milestone-1/addressbook.jar](https://github.com/jedkohjk/AB3-J17/releases/download/Milestone-1/addressbook.jar)

### CODE REVIEW

**Comment by damithc** [comment]
We faced this ARM vs Intel issue even with Java 11 but eventually we figured using the prescribed Azul JDK removed the need for adding `mac-aarch64` to the Gradle file. I'm hoping we can do something similar with Java 17. Can you try the JDK version suggestion in https://github.com/nus-oss/AB3-J17/pull/6 and report findings?

**Comment by jedkohjk** [comment]
Hi, I actually didn't have any issues with my Windows laptop. I made the changes after comparing with my friend who used mac (who is also doing this project). I will remove the additions of `mac-aarch64` from my PR.

File: build.gradle [hunk 1]
```diff
@@ -8,8 +8,8 @@ plugins {
 
 mainClassName = 'seedu.address.Main'
 
-sourceCompatibility = JavaVersion.VERSION_11
-targetCompatibility = JavaVersion.VERSION_11
+sourceCompatibility = JavaVersion.VERSION_17
+targetCompatibility = JavaVersion.VERSION_17
```
Comment (damithc) [inline build.gradle:12]: hmm... not sure why this still shows up in the changeset, as this has been updated in the master branch already. 🤔

**Comment by damithc** [comment]
Closing, as the migration was done in https://github.com/nus-oss/AB3-J17/commit/fe805d8553bbc329644371a6e732a8317ea6686f

**Pull request closed without merging**